### ISSUE TITLE
Meetings as a "proper" data model

### ISSUE THREAD

**Issue by n-st** [engelsystem/engelsystem#1329]
TL;DR: Would it be an option to have a dedicated data model for meetings, rather than re-using news entries for this purpose?

There are a few tangentially related issue regarding meetings — #1300, #1174, #228 come to mind. All of them revolve around parsing meeting information more easily — which is currently hard because meetings are just plaintext news with a boolean "is_meeting" tacked onto them.

Perhaps all of those could be addressed more easily if the Engelsystem had a separate, more purpose-oriented, way of entering and handling meetings.

At first glance, meetings will usually contain the following information:
- affected angel types 
  - (and whether to consider non-confirmed angels of that type)
- one or multiple time periods (day + start time + approximate end time; note the possible edge case of meetings crossing midnight)
- one or multiple locations (location:timeperiod usually 1:1, but possibly n:1 if too many people?)
  - locations may have an associated URL (c3nav or similar)
- description text (purpose of the meeting, addressees (only angels new to that type, or perhaps all angels of that type), what things to bring, …)
  - possibly in German and English

If the Engelsystem had a way to input and store this information in a structured manner, one could much more easily generate
- notification emails _only to the affected angel types_
- calendar feeds (possibly filtered by angel type as well)
- an in-Engelsystem overview of upcoming meetings
- collision warnings between meetings and shifts (based on one's own selected meetings and shifts)

**Comment by MyIgel** [comment]
That are great ideas but at the end its pretty complicated and needs to be implemented and maintained.

Exporting even more feeds would make the engelsystem to some kind of calendar provider which adds additional complexity that i would rather not add.
-> Imho this needs further consideration, discussion and planning before starting some implementation.
